### ISSUE TITLE
Public Profile Edit Odd UI/UX Issue

### ISSUE THREAD

**Issue by notGerryMiller** [SparkDevNetwork/Rock#5816]
### Description

It appears that the campus selector is rendering outside the block(?) when updating a profile with the Public Profile Edit. The campus selector appears to be

### Actual Behavior

The campus selector appears to be rendered outside the block(?) when updating a profile with the Public Profile Edit. We understand that this is not a person attribute, but when the block has the same background color as the page, it looks like it is just floating out by itself. 
<img width="1792" alt="Screenshot 2024-04-02 at 11 12 04" src="https://github.com/SparkDevNetwork/Rock/assets/53865308/5fba72a1-0e53-40e5-9751-699de4ba8e4a">
<img width="1196" alt="Screenshot 2024-04-02 at 11 35 09" src="https://github.com/SparkDevNetwork/Rock/assets/53865308/820370d4-7ccc-4840-98fe-9d88e7068781">
![Screenshot 2024-04-02 at 11 56 19](https://github.com/SparkDevNetwork/Rock/assets/53865308/da565b67-afb5-44b8-9d3a-d6a0ff0cc777)

### Expected Behavior

The selector should be rendered down in the same element as the rest of the form. 

### Steps to Reproduce

1. Log into the demo site and go here: https://rocksolidchurchdemo.com/MyAccount
2. Click on the Update Profile Button
3. Observe the floating drop-down menu in the upper right. 

### Issue Confirmation

- [X] Perform a search on the [Github Issues](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ARock) to see if your bug or enhancement is already reported.
- [X] Try to reproduce the problem on a fresh install or on the [demo site](https://rock.rocksolidchurchdemo.com/).

### Rock Version

15.2 and 16.2

### Client Culture Setting

en-US

**Comment by JimMichael** [comment]
We assumed this was by design, because it affects the whole Family, not just the person being updated. We made this more obvious by changing the _Campus Selector Label_ block setting to "Family Campus".
![image](https://github.com/SparkDevNetwork/Rock/assets/3903408/e04a1867-06c6-42e5-9292-e6f2e7dc53de)

**Comment by notGerryMiller** [comment]
Yeah, we believe it is by design as well. However, many of our non-technical users think that it's broken and keep sending us tickets. It doesn't look the greatest.

**Comment by nairdo** [comment]
@notGerryMiller  & @JimMichael , yes, that was an intentional change we made back in v12.3, but we'll move it down a bit and into a "Family Information" section that will look something like this (if the Show Campus Selector is enabled):

![image](https://github.com/SparkDevNetwork/Rock/assets/383807/73705f13-a454-4cba-99aa-8ce6f419af9f)
